weaviate/collection/classes.py

```diff
@@ -398,6 +419,16 @@ class _CollectionConfig:
 
 
 def _collection_config_from_json(schema: Dict[str, Any]) -> _CollectionConfig:
+    def _check_if_cref(d_type: str) -> bool:
+        return d_type[0][0].lower() == d_type[0][0]
```

Comment:
check if it is an instance of `DataType`?

Reply:
Yes, the naming here is poor

Reply:
Ah sorry, I got confused. I thought this is a check when creating the schema out of the userinput and there we could have used the type of the input. Naming is fine, please ignore